conf/bblayers.conf.sample

```diff
@@ -1,40 +1,40 @@
+# LAYER_CONF_VERSION is increased each time build/conf/bblayers.conf
```

Comment:
I'm not sure what that variable actually does. I don't see any upstream documentation about it, nor do I see a place that we reference it in our pipelines. Is there something I'm missing here?

I can bump the version number, since this is a breaking change. I'm not sure which variable name is appropriate though.

Reply:
I am similarly confused. There are lots of references to other people being confused about the same in the yocto mailing list, but I don't see a definitive answer. I guess just bump the version number for now.

Reply:
Done.